DEVELOPING.md

```diff
@@ -131,6 +131,10 @@ if *debugging {
     git push origin featurebranch
     ```
 
+6.  Create a Pull Request
+    1.  Goto https://github.com/google/go-flow-levee
+    2.  Click on the "Compare & pull request" button.
```

Comment:
There may not be a "Compare & pull request" button if the branch wasn't pushed recently :thinking: